spec/spec_helper.rb

```diff
@@ -8,6 +8,7 @@
 # in some spec runs.
 if ENV['USE_RAILS'] == 'true'
   require 'active_record'
+  require 'active_support'
```

Comment:
Probably a moot point if you've got Rails, but how about just `require 'active_support/notifications'`?